extensions/blocks/eventbrite/modal-button-preview.js

```diff
@@ -20,6 +20,7 @@ import { __ } from '@wordpress/i18n';
 import { Component, useCallback } from '@wordpress/element';
 import { compose } from '@wordpress/compose';
 import { PanelBody, RangeControl, withFallbackStyles } from '@wordpress/components';
+import { defaultModalButtonText } from '.';
```

Comment:
Is it reasonable to import this constant here, rather than add it to props?

Reply:
Yup, that's how you'd use constants! 🙂 

Just one thing: we group and comment imports by "origin".
E.g.

```es6
/**
 * External dependencies
 */
import { get } from 'lodash';

/**
 * WordPress dependencies
 */
import { __ } from '@wordpress/i18n';

/**
 * Internal dependencies
 */
import { defaultModalButtonText } from '.';
```

Sometimes, in a8c code, you might find WP deps in the same group as external deps.
I'm not sure of the reason, but Core Guten uses all three groups, so I'd keep their same format.